api/v1alpha1/helpers.go

```diff
@@ -0,0 +1,20 @@
+package v1alpha1
+
+import "k8s.io/apimachinery/pkg/util/yaml"
+
+// *****
+// Helper types
+// *****
+// Channel - list all channels for a given operator
+type Channel struct {
+	Name       string `json:"name"`
+	CurrentCSV string `json:"currentCSV"`
+}
```

Comment:
Should this be in `addonmetadata_types.go` instead?

Reply:
moved